src/data_visualization/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "data_visualization"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+chrono = "0.4.38"
+csv = "1.3.0"
+exitcode = "1.1.2"
+file = "1.1.2"
+itertools-num = "0.1.3"
+plotly = { version = "0.9", features = ["kaleido"] }
+plotly_kaleido = "0.9"
+polaris-core = "0.1.3"
+polars = { version = "0.41", features = [
+    "csv",
+    "lazy",
+    "performant",
+    "dtype-datetime",
+    "strings",
+] }
```

Comment:
will use our local `load` library to do the `csv` loading

we will remove dependencies afterwards